### ISSUE TITLE
Command-line Switches: Exit the currently running copy of NVDA if using either "-c" or "--config-path"

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#11673]
### Is your feature request related to a problem? Please describe.

Using either the "-c" or "--config-path" command-line switches allows NVDA to be run with a pre-defined configuration path - however, if running the main "nvda.exe" executable with either of these switches, it refuses to run if an existing copy of NVDA is active. Could we change this behaviour so that running NVDA with either of these switches exits the currently-running copy if found? This would make it easier to use these switches without explicitly having to define that the current copy of NVDA should be closed.

**Comment by CyrilleB79** [comment]
Could you please provide detailed steps to reproduce the issue, the expected result and the actual result? I have tried to launch "nvda" while "nvda -c aFolder" was running and vice versa. All behaved as expected for me, i.e. the new launched program replaced the running copy.

**Comment by lukaszgo1** [comment]
I also cannot reproduce this issue. Please note that before NVDA 2019.3 by default current copy of NVDA was not  closed when starting another copy, so if one of your NVDA  executable's is older this may happen.

**Comment by elliott94** [comment]
This is exactly what happened in my test environment - an older build was being used. My apologies - thank you for the clarification, will close this.
